src/main/java/uk/ac/ebi/eva/pipeline/io/writers/AnnotationMongoWriter.java

```diff
@@ -86,10 +86,12 @@ public AnnotationMongoWriter(MongoOperations mongoOperations, String collection)
     }
 
     @Override
-    public void write(List<? extends Annotation> annotations) throws Exception {
+    public void write(List<? extends List<Annotation>> annotations) throws Exception {
         BulkOperations bulk = mongoOperations.bulkOps(BulkOperations.BulkMode.UNORDERED, collection);
-        prepareBulk(annotations, bulk);
-        bulk.execute();
+        for (List<Annotation> annotationList : annotations) {
+            prepareBulk(annotationList, bulk);
+            bulk.execute();
```

Comment:
Maybe we could join all the annotation lists? Or do you think that it will be really heavy?